prediction_market_agent/tools/anvil/models.py

```
@@ -0,0 +1,96 @@
+class AgentCommunicationMessage(EventBase):
+    sender: HexBytes
+    agent_address: HexBytes = Field(alias="agentAddress")
+    message: bytes
+    value: int
+
+    @computed_field
+    @property
+    def decoded_message(self) -> str:
+        try:
+            return self.message.decode("utf-8")
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)